websites/C/cda.pl/dist/metadata.json

```diff
@@ -12,8 +12,8 @@
 		"vi_VN": "Dịch vụ cung cấp VOD độc lập lớn nhất Phần Lan. Cung cấp phim và sê-ri chất lượng cao."
 	},
 	"url": "www.cda.pl",
-	"version": "1.0.12",
-	"logo": "https://i.imgur.com/SQlMvJh.jpg",
+	"version": "2.0.0",
+	"logo": "https://i.imgur.com/AD1naGM.png",
```

Comment:
don't know where you got this logo from but according to [this page](https://www.cda.pl/logotyp) the old one is correct